src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -2164,24 +2164,17 @@ namespace DataHeatBalance {
     struct HeatReclaimDataBase
     {
         // Members
-        std::string Name;       // Name of DX Coil
-        std::string SourceType; // SourceType for DX Coil
+        std::string Name;       // Name of Coil
+        std::string SourceType; // SourceType for Coil
         Real64 AvailCapacity;   // Total available heat reclaim capacity
+        Real64 ReclaimEfficiencyTotal;   // Total reclaimed portion
+        Real64 WaterHeatingDesuperheaterReclaimedHeatTotal;    // total reclaimed heat by water heating desuperheater coils
+        Real64 HVACDesuperheaterReclaimedHeatTotal;    // total reclaimed heat by water heating desuperheater coils
+        Array1D<Real64> WaterHeatingDesuperheaterReclaimedHeat; // heat reclaimed by water heating desuperheater coils
+        Array1D<Real64> HVACDesuperheaterReclaimedHeat; // heat reclaimed by water heating desuperheater coils
```

Comment:
Added waste heat user arrays to track them separately. Use this instead of decrementing AvailCapacity to avoid repeatedly extract heat in iterations and to keep data consistent through namespaces.